### PR TITLE
IVYPORTAL-17829 Use Portal view expire exception dialog inside task iframe

### PR DESCRIPTION
**Pull request by ntloc-axonivy**

### CODE REVIEW

**Review by mnhnam-axonivy — APPROVED** [review]
lgtm

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 1]
```diff
@@ -3,18 +3,19 @@
 Error Handling
 ==============
 
-In this section, we introduce two kinds of errors, and describe when and how to
-handle them in Portal.
+In this section, we want to mentions about types of errors which user may encounters when using the Portal,
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:6]: In this section, we want to mentions about types of errors which user may encounters when using the Portal,

-->
In this section, we want to mention the types of errors that users may encounter when using the Portal

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 2]
```diff
@@ -3,18 +3,19 @@
...
+and how to enhance error handling approaches within the Portal.
 
--  Ajax error: this kind of error occurs during a JSF ajax request,
-   for example when the user clicks on the show full mode button to tell
-   the task widget to switch to full mode. Without error handling, the end user
-   would not get any form of feedback if the action has been performed successfully or not.
+-  Ajax error: These errors occur during a JSF Ajax request,
+   such as when the user clicks the "Show Full Mode" button to switch the task widget to full mode.
+   Without proper error handling, users receive no feedback on whether the action was successful.
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:11]: on whether the action was successful. -->  if the action was unsuccessful.

(no feedback needed if all is ok)

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 3]
```diff
@@ -3,18 +3,19 @@
...
 
--  Non-ajax error: this kind of errors occurs when the user accesses the Portal
-   from a URL that could not be handled successfully by the server side, or
-   being navigated by a corrupt URL. For example, when the user clicks
-   on a link to start a task that does not exist.
+-  Non-ajax error: These errors occur when a user accesses the Portal via a URL that cannot be processed successfully by the server,
+   or when navigating a corrupt URL.
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:14]: or when navigating a corrupt URL --> or when navigating to a corrupt URL

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 4]
```diff
@@ -3,18 +3,19 @@
...
+   For example, this can happen if a user clicks a link to start a task that does not exist.
+
+-  View expire error: These errors occur when a user submits a request for a page that no longer has an active view in the session,
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:17]: View expire error -->  View expired error

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 5]
```diff
@@ -96,6 +97,36 @@ Example testing URL: 500
...
+-------------------
+
+Introduction
+^^^^^^^^^^^^
+
+.. _components-error-handling-customize-view-expired-dialog:
+
+Ivy provides a warning dialog for the ``ViewExpiredException``,
+located in the ``webContent/layouts/includes/exception.xhtml`` file of your project.
+If any dialog of your project using :ref:`iframe-in-portal`, and encounters a ``ViewExpiredException``,
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:111]: If any dialog of your project using :ref:`iframe-in-portal`, 

-->

If any dialog of your project is using :ref:`iframe-in-portal`,

File: Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst [hunk 6]
```diff
@@ -96,6 +97,36 @@ Example testing URL: 500
...
+the default wanring dialog will be displayed.
```
Comment (phochstr) [inline Documentation/portal-guide/source/portal-developer-guide/components/error-handling.rst:112]: wanring --> warning

**Review by phochstr — APPROVED** [review]
There seems to be some randomness to the use of trailing s'es ;-)
Good work - correct and merge.

Nice new features!

**Pull request merged**